app/views/repos/new.html.slim

```diff
@@ -18,13 +18,13 @@
 
   .tab-content
     .tab-pane.active#own
-      = render 'shared/add_repos_list', repos: @own_repos
+      = render 'shared/add_repos_list', repos: @own_repos, all_repos: Repo.all
```

Comment:
Well, the reason I'm passing the repos is so we only need to access the db once. If we call Repo.all each time the method is called, it's eliminating any benefit and it would probably better to just go back to the original implementation. Those are just my thoughts.